src/io/flutter/FlutterMessages.java

```diff
@@ -29,6 +29,16 @@ public static void showError(String title, String message) {
                        NotificationType.ERROR));
   }
 
+  public static void showMessage(String title, String message) {
+    final Notification notification = new Notification(
+      FLUTTER_NOTIFICATION_GOUP_ID,
+      title,
+      message,
+      NotificationType.INFORMATION);
```

Comment:
I'm happy to add the info on the tools side - any preference on the serialization?